### PR TITLE
SRS _SST new dataset

### PR DESCRIPTION
**Pull request by lbesnard**
* add new SRS SST products for gogoduck by creating new views, including:
    L3S-3d day, night day&night
    L3C
    L3U
 * changes include context to only include *.nc and not nc4 files.
 * exclude all ????/RT/* realtime files.

### CODE REVIEW

**Comment by lbesnard** [comment]
databags need to be changed

**Comment by ggalibert** [comment]
I can see that sourcedir context has also changed from "/mnt/opendap/2/SRS/sst/ghrsst-fv02" to "/mnt/opendap/2/SRS/sst/ghrsst". Is this intended?

File: workspace/SRS_SST/process/SRS_SST_harvester_1.0.item [hunk 1]
```diff
@@ -108,7 +108,7 @@
     <elementParameter field="TEXT" name="PASS" value="context.Destination_Password"/>
     <elementParameter field="FILE" name="FILENAME" value="&quot;/home/mhidas/work/sql/data_harvest/anmn_ts.changelog.xml&quot;"/>
```
Comment (ggalibert) [inline workspace/SRS_SST/process/SRS_SST_harvester_1.0.item]: This is weird that some reference to Marty's ANMN_TS config is found here.

**Comment by ggalibert** [comment]
Otherwise looks good to me. Happy to merge when you reply to my comments.

**Pull request merged**